lib/manager/components/DeploymentConfirmModal.js

```diff
@@ -0,0 +1,154 @@
+  _onClickDeploy = () => {
+    const {deployment, deployToTargetClicked, target} = this.props
+    deployToTargetClicked(deployment, target)
+    this.setState({isDeployed: true})
+  }
+
+  _onClose = () => this.props.onClose()
+
+  _getFeedNames = (versions) => versions.map(v => v.feedSource.name).join(', ')
+
+  render () {
+    const {Body, Footer, Header, Title} = Modal
+    const {deployment, oldDeployment, target} = this.props
+    const {isDeployed} = this.state
+    const isMissingFeeds = deployment.feedVersions.length === 0
+    const isMissingBounds = !deployment.projectBounds
+    const {east, north, south, west} = deployment.projectBounds || {east: 10, north: 10, west: -10, south: -10}
+    const boundsTooLarge = east - west > BOUNDS_LIMIT || north - south > BOUNDS_LIMIT
+    const bounds = [[north, east], [south, west]]
+    const deploymentAreaKm = isMissingBounds
+      ? 0
+      : (area(polygon([[[north, east], [north, west], [south, west], [south, east], [north, east]]])) / (1000 * 1000)).toFixed(3)
+    const deploymentIsDisabled = isDeployed || boundsTooLarge || isMissingBounds || isMissingFeeds
+    const expiredFeeds = deployment.feedVersions.filter(versionHasExpired)
+    return (
+      <Modal show onHide={this._onClose}>
+        <Header>
+          <Title>
+            {getMessage(this.messages, 'deploy')} {deployment.name}{' '}
+            {getMessage(this.messages, 'to')} {target}?
+          </Title>
+        </Header>
+        <Body>
+          <h3>Deployment Settings</h3>
+          <ul className='list-unstyled'>
+            <li>
+              <Glyphicon glyph='fullscreen' />{' '}OSM bounds:{' '}
+              {isMissingBounds
+                ? getMessage(this.messages, 'invalidBounds')
+                : `${north.toFixed(6)}, ${east.toFixed(6)} ${getMessage(this.messages, 'to')} ${south.toFixed(6)}, ${west.toFixed(6)} (${deploymentAreaKm} sq. km)`
+              }
+            </li>
+            <li>
+              <Glyphicon glyph='list' />{' '}Deploying {deployment.feedVersions.length} feeds:{' '}
+              {this._getFeedNames(deployment.feedVersions)}
+            </li>
+          </ul>
+          <h3>OpenTripPlanner Settings</h3>
+          <ul className='list-unstyled'>
+            <li>Router ID: {deployment.routerId || '[default]'}</li>
+            <li>Build config: {deployment.customBuildConfig ? 'custom' : 'default'}</li>
+            <li>Router config: {deployment.customRouterConfig ? 'custom' : 'default'}</li>
+          </ul>
+          <Map
+            ref='map'
+            bounds={bounds}
+            scrollWheelZoom={false}
+            style={{width: '100%', height: '300px'}}>
+            <TileLayer
+              url={defaultTileURL()}
+              attribution={process.env.MAPBOX_ATTRIBUTION} />
+            {!isMissingBounds &&
+              <Rectangle
+                bounds={bounds}
+                fillOpacity={0} />
+            }
+          </Map>
+          {isMissingFeeds
```

Comment:
It may be better to unchain these and instead show all at once.  ie:

```
{isMissingFeeds && <Alert...
{isMissingBounds && <Alert...
{boundsToLarge && <Alert...
```